### PR TITLE
user manager 1

### PR DESCRIPTION
**Pull request by 2751997nam**
user manager feature 1

### CODE REVIEW

**Comment by 2751997nam** [comment]
ready

**Comment by BunnyPi04** [comment]
commented

File: app/Models/User.php [hunk 1]
```diff
@@ -81,13 +82,38 @@ public function provider_users()
...
+    public function scopeBlocked($query)
+    {
+        return $query->where('status', '=', '-1');
+    }
+
+    public function scopeOption($query, $option) {
+        if($option === 'verified') return $this->scopeVerified($query);
+        if($option === 'unverify') return $this->scopeUnverify($query);
+        if($option === 'blocked') return $this->scopeBlocked($query);
+        return $query;
```
Comment (BunnyPi04) [inline app/Models/User.php]: thiếu dòng trắng trước return

**Comment by BunnyPi04** [comment]
commented

**Comment by 2751997nam** [comment]
ready

**Pull request merged**